From d2e49c9904897d418930d2e014ad1c4d86df93c4 Mon Sep 17 00:00:00 2001
From: David Edmundson <david@davidedmundson.co.uk>
Date: Sun, 16 Jun 2024 21:30:28 +0000
Subject: [PATCH] Ensure WinPR version matches FreeRDP version

Mixing WinPR 3 with FreeRDP 2 results in disaster.

See also: https://invent.kde.org/network/krdc/-/merge_requests/90
Downstream report: https://bugs.gentoo.org/934387

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>


(cherry picked from commit e19742ddb3d7dd99892a9e1bd87bd615ae0c059d)

Co-authored-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5c39ba..c23be53 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,8 +41,8 @@ find_package(KF6 ${KF6_MIN_VERSION} REQUIRED COMPONENTS Config DBusAddons KCMUti
 find_package(KF6 ${KF6_MIN_VERSION} REQUIRED COMPONENTS Config DBusAddons)
 
 find_package(FreeRDP 2.10 REQUIRED)
-find_package(WinPR REQUIRED)
-find_package(FreeRDP-Server REQUIRED)
+find_package(WinPR 2.10 REQUIRED)
+find_package(FreeRDP-Server 2.10 REQUIRED)
 find_package(KPipeWire 5.27.80 REQUIRED)
 find_package(XKB REQUIRED)
 
-- 
GitLab

